Coalition Folklore: Kimek (#5870)
endless-sky/endless-sky: data/coalition/coalition missions.txt

				accept
			label noship
			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`

mission "Coalition Folklore: Kimek"
	minor
	invisible
	to offer
		random < 30
	source
		attributes "kimek" "folklore"
	on offer
		conversation
			`A Kimek flags you down when you enter the spaceport. "Busy are you, Captain?" He asks. "Taking place today, a cultural festival is, and participate in it, anyone can. A reenactment of the old 'Spring Siege' folk tale, it is. Play the part of one of the characters, anyone can."`
			choice

> 				`	"That sounds like fun, where is it being held?"`

Reviewer: ```suggestion
				`	"That sounds like fun Where is it being held?"`
```

				`	"Sorry, I'm too busy right now for that."`
					decline

> 			`	He asks you to follow him, and you two board a train leading to another section of the city. You get off at the edge of a large park, where may sets of camps dot the savanna plains, and an equal number of wooden forts stare down from the top of a long hill. Each "set" of fort and camp is separated from the others by many patches of vegetation that go up the hill.`

Reviewer 2: ```suggestion
			`	He asks you to follow him, and you two board a train leading to another section of the city. You get off at the edge of a large park, where many sets of camps dot the savanna plains, and an equal number of wooden forts stare down from the top of a long hill. Each "set" of fort and camp is separated from the others by many patches of vegetation that go up the hill.`
```

> 			`	You follow the Kimek for a while longer, until he finds you a pairing of camp and fort that have a vacant spot in either side. "Well Captain, choose a side to play as, you must. The hang of it, you will get, once started, the event has, but as a quick summary, take the 'crops' inside the fort before time runs out, the attacking team must. Prevent them from doing that, the defenders must. At the end, rewarded, the victors are."`

Reviewer: ```suggestion
			`	You follow the Kimek for a while longer until he finds you a pairing of camp and fort that have a vacant spot in either side. "Well Captain, choose a side to play as, you must. The hang of it, you will get, once started, the event has, but as a quick summary, take the 'crops' inside the fort before time runs out, the attacking team must. Prevent them from doing that, the defenders must. At the end, rewarded, the victors are."`
```

			choice
				`	"I'll join the attackers."`
					goto att
				`	"I'll join the defenders."`
					goto def
				`	"Hang on, if this is a folk tale, how come you're reenacting it as if any side could win?"`

> 			`	He chitters, as if expecting the question. "Witnessed by a minstrel, the battle was. Travelling on fields like these, he was, before the fort, and the raider camp, he spotted in the distance. Hid himself in foliage close to the scene, he had, and writing and painting it as it happened, he was, but when reached its climax, the battle had, swept up by the wind, many flowers were, and block his vision they did. Once settled down the wind had, over, the battle was Spring it was, so 'Spring Siege' became the story's title."`

Reviewer 2: ```suggestion
			`	He chitters, as if expecting the question. "Witnessed by a minstrel, the battle was. Travelling on fields like these, he was, before the fort, and the raider camp, he spotted in the distance. Hid himself in foliage close to the scene, he had, and writing and painting it as it happened, he was, but when reached its climax, the battle had, swept up by the wind, many flowers were, and block his vision they did. Once settled down the wind had, over, the battle was. Spring it was, so 'Spring Siege' became the story's title."`
```

> 			`	He chitters, as if expecting the question. "Witnessed by a minstrel, the battle was. Travelling on fields like these, he was, before the fort, and the raider camp, he spotted in the distance. Hid himself in foliage close to the scene, he had, and writing and painting it as it happened, he was, but when reached its climax, the battle had, swept up by the wind, many flowers were, and block his vision they did. Once settled down the wind had, over, the battle was. Spring it was, so 'Spring Siege' became the story's title."`

Reviewer: ```suggestion
			`	He chitters, as if expecting the question. "Witnessed by a minstrel, the battle was. Travelling on fields like these, he was, before the fort and the raider camp, he spotted in the distance. Hid himself in foliage close to the scene, he had, and writing and painting it as it happened, he was. When reached its climax, the battle had, swept up by the wind, many flowers were, and block his vision they did. Once settled down the wind had, over, the battle was. Spring it was, so 'Spring Siege' became the story's title."`
```

> 			`	He chitters, as if expecting the question. "Witnessed by a minstrel, the battle was. Travelling on fields like these, he was, before the fort and the raider camp, he spotted in the distance. Hid himself in foliage close to the scene, he had, and writing and painting it as it happened, he was. When reached its climax, the battle had, swept up by the wind, many flowers were, and block his vision they did. Once settled down the wind had, over, the battle was. Spring it was, so 'Spring Siege' became the story's title."`

Reviewer 3: ```suggestion
			`	He chitters, as if expecting the question. "Witnessed by a minstrel, the battle was. Traveling on fields like these, he was, before the fort and the raider camp, he spotted in the distance. Hid himself in foliage close to the scene, he had, and writing and painting it as it happened, he was. When reached its climax, the battle had, swept up by the wind, many flowers were, and block his vision they did. Once settled down the wind had, over, the battle was. Spring it was, so 'Spring Siege' became the story's title."`
```

> 			`	Seeing you have more questions now, he continues. "Silly and nonsensical at times, the story is. Know that, we do, and that simply looking to entertain a crowd, the minstrel was, but with all forms of classic tales, is that not the case?" He chitters some more, and looks at the group up on the hill, and the other in the camp, before continuing. "Foreign as it might seem, participate in it, will you not? Perhaps like it, you might."`

Reviewer: ```suggestion
			`	Seeing you have more questions now, he continues. "Silly and nonsensical at times, the story is. Know that, we do, and that simply looking to entertain a crowd, the minstrel was, but with all forms of classic tales, is that not the case?" He chitters some more, and looks at the group up on the hill and the other in the camp, before continuing. "Foreign as it might seem, participate in it, will you not? Perhaps like it, you might."`
```

			choice
				`	"Alright, I'll join the attackers."`
					goto att
				`	"Alright, I'll join the defenders."`
			label def

> 			`	He brings you to the small fort up the hill, its gates open as those inside, mostly Kimek as expected, but also a couple Arachi, and a lone Saryd, are still getting ready. A small "farm" sits at the middle, with plastic replicas serving as the crops. The walls of the fort are made of various tall logs, covered by dense patches of reddish thorns, and inside you can see a higher level with battlements is accessible via staircases on either side of the courtyard. You also notice that the defenders are all wearing a green strip of cloth over their clothing. Another Kimek organizer comes to meet you two at the gate, wearing a rugged leather vest that you assume is the Kimek equivalent of medieval armor. He greets his colleague, who introduces you and tells him you chose to join the defenders, before heading back down the hill.`

Reviewer 4: ```suggestion
			`	He brings you to the small fort up the hill, its gates open as those inside, mostly Kimek as expected, but also a couple Arachi, and a lone Saryd, are still getting ready. A small "farm" sits at the middle, with plastic replicas serving as the crops. The walls of the fort are made of various tall logs, densely covered by closely-spaced tufts of long and vicious, reddish thorns nearly as long as your arm. Inside you can see a higher level with battlements is accessible via staircases on either side of the courtyard. You also notice that the defenders are all wearing a green strip of cloth over their clothing. Another Kimek organizer comes to meet you two at the gate, wearing a rugged leather vest that you assume is the Kimek equivalent of medieval armor. He greets his colleague, who introduces you and tells him you chose to join the defenders, before heading back down the hill.`
```

> 			`	He brings you to the small fort up the hill. Its gates open as those inside (mostly Kimek as expected, but also a couple Arachi and a lone Saryd) are still getting ready. A small "farm" sits at the middle, with plastic replicas serving as the crops. The walls of the fort are made of various tall logs, densely covered by tufts of vicious, reddish thorns nearly as long as your arm. Inside you can see that a higher level with battlements is accessible via staircases on either side of the courtyard; you also notice that the defenders are all wearing a green strip of cloth over their clothing. Another Kimek organizer comes to meet you two at the gate, wearing a rugged leather vest that you assume is the Kimek equivalent of medieval armor. He greets his colleague, who introduces you and tells him you chose to join the defenders, before heading back down the hill.`

Reviewer: ```suggestion
			`	He brings you to the small fort up the hill. Its gates open as those inside (mostly Kimek as expected, but also a couple Arachi and a lone Saryd) are still getting ready. A small "farm" sits at the middle, with plastic replicas serving as the crops. The walls of the fort are made of various tall logs, densely covered by tufts of vicious, reddish thorns nearly as long as your arm. Inside, you can see that a higher level with battlements is accessible via staircases on either side of the courtyard; you also notice that the defenders are all wearing a green strip of cloth over their clothing. Another Kimek organizer, wearing a rugged leather vest that you assume is the Kimek equivalent of medieval armor, comes to meet you two at the gate. He greets his colleague, who introduces you and tells him you chose to join the defenders, before heading back down the hill.`
```
Just a comma fix and a switch of the order of phrases in one sentence.

> 			`	He brings you to the small fort up the hill. Its gates open as those inside (mostly Kimek as expected, but also a couple Arachi and a lone Saryd) are still getting ready. A small "farm" sits at the middle, with plastic replicas serving as the crops. The walls of the fort are made of various tall logs, densely covered by tufts of vicious, reddish thorns nearly as long as your arm. Inside, you can see that a higher level with battlements is accessible via staircases on either side of the courtyard; you also notice that the defenders are all wearing a green strip of cloth over their clothing. Another Kimek organizer, wearing a rugged leather vest that you assume is the Kimek equivalent of medieval armor, comes to meet you both at the gate. He greets his colleague, who introduces you and says that you have chosen to join the defenders, before heading back down the hill.`

Reviewer 3: ```suggestion
			`	He brings you to the small fort up the hill. Its gates open as those inside (mostly Kimek as expected, but also a couple Arachi and a lone Saryd) are still getting ready. A small "farm" sits in the middle, with plastic replicas serving as the crops. The walls of the fort are made of various tall logs, densely covered by tufts of vicious, reddish thorns nearly as long as your arm. Inside, you can see that a higher level with battlements is accessible via staircases on either side of the courtyard; you also notice that the defenders are all wearing a green strip of cloth over their clothing. Another Kimek organizer, wearing a rugged leather vest that you assume is the Kimek equivalent of medieval armor, comes to meet you both at the gate. He greets his colleague, who introduces you and says that you have chosen to join the defenders, before heading back down the hill.`
```

> 			`	"Glad to have you as part of the garrison, our fort is," the armored Kimek begins. "Told about the enemy's objective, I assume you have been. Protect our farm, we must, so allow the produce to be stolen, we cannot." He tells you to follow him to the 'keep,' a shack too small for any non-Kimek, and gives you a long piece of green cloth. He then informs you that it is meant to be worn, so that others can tell what team you are on. As you wrap it around you like a scarf, you notice that, behind the shack, there is a smaller gate that grants entry to the fort. Once you're done, he gives you a brief tour of the rest of the fort, taking you up the battlements and explaining that, above the main gate, they have buckets of a water-like, slippery liquid. During the re-enactment, they are poured downhill and used to slow the attackers.`

Reviewer: ```suggestion
			`	"Glad to have you as part of the garrison, our fort is," the armored Kimek begins. "Told about the enemy's objective, I assume you have been. Protect our farm, we must, so allow the produce to be stolen, we cannot." He tells you to follow him to the "keep," a shack too small for any non-Kimek, and gives you a long piece of green cloth. He then informs you that it is meant to be worn, so that others can tell what team you are on. As you wrap it around you like a scarf, you notice that, behind the shack, there is a smaller gate that grants entry to the fort. Once you're done, he gives you a brief tour of the rest of the fort, taking you up the battlements and explaining that, above the main gate, they have buckets of a water-like, slippery liquid. During the re-enactment, they are poured downhill and used to slow the attackers.`
```

> 			`	Once he's done, he tells you to wait in the courtyard, as the "battle" will begin soon. A few minutes pass before a loud whistle blows, and the Kimek in charge of the fort tells everyone that the other side has begun their attack. The front gate is closed and some of the defenders are looking for heavy objects to barricade it; others are up by the battlements, waiting for the right time to use the buckets with the slippery liquid.`

Reviewer: ```suggestion
			`	Once he's done, he tells you to wait in the courtyard, as the "battle" will begin soon. A few minutes pass before a loud whistle blows, and the Kimek in charge of the fort tells everyone that the other side has begun their attack. The front gate is closed, and some of the defenders placing heavy objects behind to barricade it; others are up by the battlements, waiting for the right time to use the buckets with the slippery liquid.`
```
Another comma fix, and a small edit: if they've spent some time waiting for the battle to start, wouldn't they have already located things to barricade with instead of just starting to look when the fight begins?

			choice
				`	(Help reinforce the main gate.)`
					goto defr
				`	(Go help slow down the attackers.)`
					goto defw
			label defr

> 			`	You join the others in looking for heavy objects, finding some crates behind the small shack, which the Arachi bring to the gate, and then go some Kimek take some of the logs from one of the staircases, burying one end in the ground, while the other touch the gate. The liquid poured on the attackers did slow them down for some minutes, but once you're done they're already banging at the gate. Most of those inside the fort, including the Saryd, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`

Reviewer 2: ```suggestion
			`	You join the others in looking for heavy objects, finding some crates behind the small shack, which the Arachi bring to the gate, and then some Kimek take some of the logs from one of the staircases, burying one end in the ground, while the other touches the gate. The liquid poured on the attackers did slow them down for some minutes, but once you're done they're already banging at the gate. Most of those inside the fort, including the Saryd, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`
```

> 			`	You join the others in looking for heavy objects, finding some crates behind the small shack, which the Arachi bring to the gate, and then some Kimek take some of the logs from one of the staircases, burying one end in the ground, while the other touches the gate. The liquid poured on the attackers did slow them down for some minutes, but once you're done they're already banging at the gate. Most of those inside the fort, including the Saryd, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`

Author: As I said, I want the folklore bits to feel distinct, so I thought of making the Kimek one a reenactment, where the player actually participates in it.
If they'll let a human participate, it is only natural they already allow for members of the other two Coalition species to participate as well. As for the roles bit, that wasn't intentional, I just had those in to show some members of the teams were better at certain parts because of their different alien bodies.
I was originally going to have a bit where they explain to you that they've incorporated the other two Coalition species' characteristics in the rules of this and all, but I didn't know where to put it, and felt it could bloat things unnecessarily.

> 			`	You join the others in looking for heavy objects, finding some crates behind the small shack, which the Arachi bring to the gate, and then some Kimek take some of the logs from one of the staircases, burying one end in the ground, while the other touches the gate. The liquid poured on the attackers did slow them down for some minutes, but once you're done they're already banging at the gate. Most of those inside the fort, including the Saryd, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`

Reviewer 4: This could maybe be helped then by replacing group references with participatory indications then, so as not to bloat but still to avoid giving explicit roles. My suggestion also contains some grammar tweaking.
```suggestion
			`	You join the others in looking for heavy objects, finding some crates behind the small shack, which a couple of Arach help bring to the gate. Some Kimek take logs stashed under a staircase and bring them as well, burying one end in the ground, while the other touches the gate. The liquid poured on the attackers does slow them down for some minutes, but once you're done they're already banging at the gate. Most of those inside the fort, particularly the Saryd that have joined in, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`
```

			choice
				`	(Help keep the main gate closed.)`
					goto defrw

> 				`	(Go with the others check the back gate.)`

Reviewer 2: ```suggestion
				`	(Go with the others and check the back gate.)`
```

> 			`	You join the others in looking for heavy objects, finding some crates behind the small shack, which a couple of Arach help bring to the gate. Some Kimek take logs stashed under a staircase and bring them as well, burying one end in the ground with the other end against the door. The liquid poured on the attackers does slow them down for some minutes, but once you're done they're already banging at the gate. Most of those inside the fort, particularly the Saryd that has joined in, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`

Reviewer: ```suggestion
			`	You join the others in moving heavy objects, finding some crates behind the small shack, which a couple of Arach help bring to the gate. Some Kimek take logs stashed under a staircase and bring them as well, burying one end in the ground with the other end against the door. The liquid poured on the attackers does slow them down for some minutes, but once you're done, they're already banging at the gate. Most of those inside the fort, particularly the Saryd that has joined in, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`
```

> 			`	You join the others in looking for heavy objects, finding some crates behind the small shack, which a couple of Arach help bring to the gate. Some Kimek take logs stashed under a staircase and bring them as well, burying one end in the ground with the other end against the door. The liquid poured on the attackers does slow them down for some minutes, but once you're done they're already banging at the gate. Most of those inside the fort, particularly the Saryd that have joined in, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`

Reviewer 5: ```suggestion
			`	You join the others in looking for heavy objects, finding some crates behind the small shack, which a couple of Arach help bring to the gate. Some Kimek take logs stashed under a staircase and bring them as well, burying one end in the ground with the other end against the door. The liquid poured on the attackers does slow them down for some minutes, but once you're done they're already banging at the gate. Most of those inside the fort, particularly the Saryd that has joined in, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`
```

> 			`	You join the others in gathering and moving heavy objects, and quickly find some crates behind the small shack, which a couple of Arach help bring to the gate. Some Kimek take logs they find stashed under a staircase and bring them as well, burying one end in the ground with the other end against the door. The liquid poured on the attackers does slow them down for some minutes, but once you're done, they're already banging at the gate. Most of those inside the fort, particularly the Saryd that has joined in, head to the main gate to try and keep it closed, but a few Kimek head around the back to check on the smaller gate.`

Reviewer 3: ```suggestion
			`	You join the others in gathering and moving heavy objects, and quickly find some crates behind the small shack, which a couple of Arachi help bring to the gate. Some Kimek take logs they find stashed under a staircase and bring them as well, burying one end in the ground with the other end against the door. The liquid poured on the attackers does slow them down for some minutes, but once you're done, they're already banging at the gate. Most of those inside the fort, particularly the Saryd that has joined in, head to the main gate to try and keep it closed, but a few Kimek head around the back to check on the smaller gate.`
```

> 			`	Once you get to the back gate, you and the Kimek there stay quiet and try to listen for anyone approaching this side of the fort. Sure enough, you hear scuttling Kimek legs swishing through the grass, as the attackers come circling around. Fortunately, not many came to this gate, so you and the Kimek don't struggle much to keep it closed. Thanks to the main gate having been reinforced, the group there holds well enough that some more come to the back to help, and after some minutes of struggling with the attackers on the gate, another whistle is sounded. It marks that time has run out and that the defenders have won.`

Reviewer: ```suggestion
			`	Once you get to the back gate, you and the Kimek there stay quiet and try to listen for anyone approaching this side of the fort. Sure enough, you hear scuttling Kimek legs swishing through the grass as the attackers come circling around. Fortunately, not many came to this gate, so you and the Kimek don't struggle much to keep it closed. Thanks to the main gate having been reinforced, the group there holds well enough that some more come to the back to help, and after some minutes of struggling with the attackers on the gate, another whistle is sounded. It marks that time has run out and that the defenders have won.`
```

				goto won
			label defrw

> 			`	With the fortifications from earlier helping keep it shut, holding the main gate proves easy enough. After a few minutes pass, however, you hear a commotion from behind the shack. As you and others go there to check, you arrive to see the small gate open, the attackers having made their way inside. The whistle sounds again, marking the end of the battle, and the victory of the attackers.`

Reviewer: ```suggestion
			`	With the fortifications from earlier helping keep it shut, holding the main gate proves easy enough. After a few minutes pass, however, you hear a commotion from behind the shack. As you and others go there to check, you arrive to see the small gate open, the attackers having made their way inside. The whistle sounds again, marking the end of the battle and the victory of the attackers.`
```

				goto lost
			label defw

> 			`	You go up the stairs, and are given one of the buckets with the slippery substance. You notice that the others up here are all Kimek and Arachi, however, and they are only pouring the liquid down at the front. When you see some attackers advancing from the sides, where the liquid hasn't been poured, you chuck the bucket away. This leaves a line of the liquid on one side, making those attacking from there lose their footing and trip again and again. Once your teammates see what you did, they egg you on and encourage you to throw more buckets to the sides as they continue to pour down the middle. Despite your efforts, however, the attackers eventually make it up the hill and start banging at the gate. Most of those inside the fort, including the Saryd, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`

Reviewer: ```suggestion
			`	You go up the stairs and are given one of the buckets with the slippery substance. You notice that the others up here are all Kimek and Arachi, however, and they are only pouring the liquid down at the front. When you see some attackers advancing from the sides, where the liquid hasn't been poured, you chuck the bucket away. This leaves a line of the liquid on one side, making those attacking from there lose their footing and trip again and again. Once your teammates see what you did, they egg you on and encourage you to throw more buckets to the sides as they continue to pour down the middle. Despite your efforts, however, the attackers eventually make it up the hill and start banging at the gate. Most of those inside the fort, including the Saryd, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`
```

> 			`	You go up the stairs and are given one of the buckets with the slippery substance. You notice that the others up here are all Kimek and Arachi, however, and they are only pouring the liquid down directly in front of the gate. When you see some of the blue sash wearing attackers advancing from the sides, where the liquid hasn't been poured, you chuck a bucket their way. This leaves a line of the liquid on that side, and those those attacking from there lose their footing, tripping and falling again and again. Once your teammates realize what you've done, they begin to egg you on, encouraging you to throw more buckets to cover the sides while they continue to drench the middle. Despite your efforts, however, the attackers do eventually make it up the hill and start banging at the gate. Most of those inside the fort, including the Saryd, head to the main gate to try and keep it closed, but some Kimek instead head around the back to check on the smaller gate.`

Reviewer 3: ```suggestion
			`	You go up the stairs and are given one of the buckets with the slippery substance. You notice that the others up here are all Kimek and Arachi, however, and they are only pouring the liquid down directly in front of the gate. When you see some of the blue sash wearing attackers advancing from the sides, where the liquid hasn't been poured, you chuck a bucket their way. This leaves a line of the liquid on that side, and those attacking from there lose their footing, tripping and falling again and again. Once your teammates realize what you've done, they begin to egg you on, encouraging you to throw more buckets to cover the sides while they continue to drench the middle. Despite your efforts, however, the attackers do eventually make it up the hill and start banging at the gate. Most of those inside the fort, including the Saryd, head to the main gate to try and keep it closed, but some Kimek instead head around the back to check on the smaller gate.`
```

			choice
				`	(Help keep the main gate closed.)`
					goto defww
				`	(Go with the others and check the back gate.)`

> 			`	Once you get to the back gate, you and the Kimek there stay quiet and try to listen for anyone approaching the fort. Sure enough, you hear scuttling Kimek legs swishing through the grass, as the attackers come circling around. Fortunately, not many came to this gate, so you and the Kimek don't struggle much to keep it closed. The group at the front gate isn't so lucky, however, and after a brief minute the whistle sounds again. Your group comes back to the courtyard to find that the gate has been busted open and that the attackers have won.`

Reviewer: ```suggestion
			`	Once you get to the back gate, you and the Kimek there stay quiet and try to listen for anyone approaching the fort. Sure enough, you hear scuttling Kimek legs swishing through the grass as the attackers come circling around. Fortunately, not many came to this gate, so you and the Kimek don't struggle much to keep it closed. The group at the front gate isn't so lucky, however, and after a brief minute, the whistle sounds again. Your group comes back to the courtyard to find that the gate has been busted open and that the attackers have won.`
```

				goto lost
			label defww
			`	At first, holding the gate isn't too troublesome, but after a few minutes pass, it starts being pounded by some form of battering ram, and before long the group there can't hold it closed. At the same time, you hear a commotion from behind the shack, and as you and others go there to check, you arrive to see the small gate open. The attackers having made their way inside via both gates, the whistle sounds again, marking the end of the battle, and the victory of the attackers.`
				goto lost
			label att

> 			`	He brings you to the nearby camp, where many Kimek, Arachi, and even a few Saryds are gathered for the event, waiting in anticipation inside one of the tents, or looking up at the fort. All of them have long, blue strips of cloth over their own clothing. Once she notices you two, another Kimek organizer, donning what looks like a thin gambeson, comes greet you two. She greets her colleague, who introduces you and tells her you chose to join the attackers, before leaving the camp to greet a family who has come to watch the event.`

Reviewer 2: ```suggestion
			`	He brings you to the nearby camp, where many Kimek, Arachi, and even a few Saryds are gathered for the event, waiting in anticipation inside one of the tents, or looking up at the fort. All of them have long, blue strips of cloth over their own clothing. Once she notices you two, another Kimek organizer, donning what looks like a thin gambeson, comes greet you two. She greets her colleague, who introduces you and tells her you chose to join the attackers, before leaving the camp to welcome a family who has come to watch the event.`
```

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow here to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth, telling you to wrap it around you, to mark what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set," marked by the patches of vegetation, telling you that you're not allowed to go past them to not disrupt other sets. "Also, against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Author: In the Coalition Games PR, most of what the Kimek do hinges on them being able of crawling up walls, so I'm not of mind to remove that ability from them.

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow here to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth, telling you to wrap it around you, to mark what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set," marked by the patches of vegetation, telling you that you're not allowed to go past them to not disrupt other sets. "Also, against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Reviewer 4: Ah, then this does present something of an issue, because walls really make no sense as a defence emplacement if that is the case.

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow here to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth, telling you to wrap it around you, to mark what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set," marked by the patches of vegetation, telling you that you're not allowed to go past them to not disrupt other sets. "Also, against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Author: That is pretty much acnowledged by the Kimek attacker leader, and if you ask the Kimek that offers you the mission about it being reenacted, he'll spell it out that it's not something meant to make perfect sense.
Such is the case for the Saryd stories, too. A moving tree saves a boy from a storm that appears to be hunting him? A whale flies up to the skies to save a fisherman from the waves? A huntress doesn't realize a gigantic beast is right behind her, and is saved by the bird she was hunting for some reason?
These folklore missions aren't meant to give detailed historical or  archeological information on any of the species, just provide an outlet to share some of their culture.

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow here to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth, telling you to wrap it around you, to mark what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set," marked by the patches of vegetation, telling you that you're not allowed to go past them to not disrupt other sets. "Also, against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Reviewer 4: Of course not, no I understand that, but there are certain underlying principles to folklore. 

Moving trees, flying fish, talking spirits, visions in water, etc. take your pick of possible things, the fact that they're handwavy myth elements is not an issue. They exist to account for the metaphor of a story that might otherwise not be very interesting, or are simply things added to account for gaps left in tall tales told by previous generations.

Some constants remain though. A house in a folk tale still has walls and a roof, even if the house grows legs and walks around. A road may be magical, made of yellow bricks, and lead to wherever it needs to go, but it is still a paved path from one place to another.

Defensive emplacements exist to act as force multipliers, they work for us because we're bad at climbing flat surfaces. Whereas if Kimek can climb them fairly easily, then the simple _concept_ of using a wall defensively would never have entered the society in the first place. So it wouldn't have ended up in folklore.

However, I have a thought that might help.
Humans migrated to locations we weren't well suited for on earth very early in our history, and we developed clothing way back even before fire. Technological adaptations to overcome our physical limitations are many. Perhaps the Kimek are indeed bad at climbing but developed some sort of foot wear that allowed them to climb very effectively at the same time that they started building cities to be taller, in much the same way that we developed bicycles to get around more quickly when our cities started growing to be bigger than one could reasonably walk across?

This way your mythos folk tale could simply be the equivalent of a pre-industrial fairytale, from before they developed the Kimek equivalent of suction shoes or something?

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow here to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth, telling you to wrap it around you, to mark what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set," marked by the patches of vegetation, telling you that you're not allowed to go past them to not disrupt other sets. "Also, against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Reviewer 5: Perhaps Kimek are only able to climb on some surfaces and not others, and the defensive walls are made of such a substance?

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow here to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth, telling you to wrap it around you, to mark what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set," marked by the patches of vegetation, telling you that you're not allowed to go past them to not disrupt other sets. "Also, against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Reviewer 4: Perhaps, but for it to work as a folk-lore type thing that material would have to be; natural (wood, stone, etc.), workable with rudimentary tools (like, hack it with a rock and tie it together with string or gravity), and abundant on their world.

Very smooth things like glass or plastic are the obvious surfaces for not being climbable, but for the concept of using a wall defensively to enter into the pre-industrial consciousness it needs to be able to be made of something you'd have in at least moderate abundance with pre-industrial technologies. I don't have a good answer for what that might be.

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow here to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth, telling you to wrap it around you, to mark what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set," marked by the patches of vegetation, telling you that you're not allowed to go past them to not disrupt other sets. "Also, against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Reviewer 2: > Ah, then this does present something of an issue, because walls really make no sense as a defence emplacement if that is the case.

I agree with walls being less useful, but they still offer some advantages that could probably justify their usage. Like blocking sight and offering a better overview for the defenders, both valuable tactical assets. Why the attackers would prefer to enter through gates might still be a problem, but I guess that could be explained by the folklore nature and the fact that every species is allowed to participate.

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow here to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth, telling you to wrap it around you, to mark what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set," marked by the patches of vegetation, telling you that you're not allowed to go past them to not disrupt other sets. "Also, against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Reviewer 4: You can block sight with a moderate mound of dirt, which is dramatically easier to build, won't be destroyed or damaged, and can still provide much of the same defensive advantage in melee. A tower provides a comparatively superior overview, and would be vastly more useful than a wall for that purpose. Especially since the Kimek aren't exactly equipped physically to use low-tech ranged weapons so being exposed on a tower isn't such a hazard. (Also, a safe place to gain advantage using ranged weapons is a big driver for the existence of better walls, which again has no bearing on the Kimek.)

Like, constraining the method of attack for the game to fit the story is fine. There's still fundamentally nothing that a wall does that isn't done better and easier by something else if you're capable of readily climbing it - which means the mere concept of using an actual high wall _as a defence_ simply doesn't make sense. It would've never entered their species' minds until they encountered alien cultures after achieving spaceflight, which is too late for it to be folklore.

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow here to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth, telling you to wrap it around you, to mark what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set," marked by the patches of vegetation, telling you that you're not allowed to go past them to not disrupt other sets. "Also, against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Author: Actually I just remembered I was supposed to have added a part saying how the outside of the walls also has wooden spikes all over, not the type that could actually hurt somebody in this case but just to make climbing them not worth it.
Would that be alright?

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow here to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth, telling you to wrap it around you, to mark what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set," marked by the patches of vegetation, telling you that you're not allowed to go past them to not disrupt other sets. "Also, against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Reviewer 4: Rather than those spikes being manually added (which is an intensive construction effort and difficult to maintain as they weather), could it instead be that the tree they use for construction naturally grows downward slanted thorns that prevent climbing? (Like big thorns, half a metre or more in length. Or vicious thickety thorns like these:)
![image](https://user-images.githubusercontent.com/60949828/115641152-ac1d0f80-a357-11eb-8bc7-db5102af56f4.png)

This just occurred to me, and if all you have to do is take one out and substitute a new one in when/if the thorns get damaged beyond use, then as a natural, renewable, and classically available resource this could justify how they came to this development.

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow here to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth, telling you to wrap it around you, to mark what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set," marked by the patches of vegetation, telling you that you're not allowed to go past them to not disrupt other sets. "Also, against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Author: Didn't know something like that even existed.
Sure, can be those.

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow her to the "war tent," one that's slightly larger than the other ones around, and gives you a long piece of blue cloth. She then informs you that it is meant to be worn so that others can tell what team you are on. Wrapping it like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set". It is marked by patches of vegetation, meant to indicate the boundaries of the game. "Also, discourage anyone from trying it, the thorns should, but against the rules, it is, to crawl up the walls of the fort, so-" She pauses, looking at you as if just realizing that such a thing is impossible for you. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do; few Saryds here I've met. Even fewer humans, as imagine, you can."`

Author: GitHub does that sometimes.

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow her to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth. She then informs you that it is meant to be worn, so that others can tell what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set". It is marked by patches of vegetation, meant to indicate the boundaries of the game. "Also, discourage anyone from trying it, the thorns should, but against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if just realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Reviewer 5: ```suggestion
			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow her to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth. She then informs you that it is meant to be worn, so that others can tell what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set". It is marked by patches of vegetation, meant to indicate the boundaries of the game. "Also, discourage anyone from trying it, the thorns should, but against the rules, it is, to crawl up the walls of the fort, so-" She pauses, looking at you as if just realizing that such a thing is impossible for you. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`
```

> 			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow her to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth. She then informs you that it is meant to be worn, so that others can tell what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set". It is marked by patches of vegetation, meant to indicate the boundaries of the game. "Also, discourage anyone from trying it, the thorns should, but against the rules, it is, to crawl up the walls of the fort, so-" She pauses, looking at you as if just realizing that such a thing is impossible for you. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`

Reviewer: ```suggestion
			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow her to the "war tent," one that's slightly larger than the other ones around, and gives you a long piece of blue cloth. She then informs you that it is meant to be worn so that others can tell what team you are on. Wrapping it like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set". It is marked by patches of vegetation, meant to indicate the boundaries of the game. "Also, discourage anyone from trying it, the thorns should, but against the rules, it is, to crawl up the walls of the fort, so-" She pauses, looking at you as if just realizing that such a thing is impossible for you. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do; few Saryds here I've met. Even fewer humans, as imagine, you can."`
```
Removed an `around` to avoid two in one sentence. Added double quotation marks to `war tent`. Made the comma after `groundlocked people do` a semicolon.

> 			`	Once she's done, she says you're free to roam the camp as you wait for the "battle" to begin. A few minutes pass before a loud whistle blows, and the Kimek in charge of the camp tells everyone that it's begun. She leads the charge up the hill, with most of those from the camp following her. Others split into smaller groups and get closer to the trees that separate the sets, trying to approach the fort from the sides.`

Reviewer 6: ```suggestion
			`	Once she's done, she says you're free to roam the camp as you wait for the "battle" to begin. A few minutes pass before a loud whistle blows, and the Kimek in charge of the camp tells everyone that the time has come. She leads the charge up the hill, with most of those from the camp following her. Others split into smaller groups and instead move up along the trees that separate the sets, trying to approach the fort from the sides.`
```

			choice
				`	(Charge with the main group.)`
					goto attw
				`	(Take the longer side route.)`
					goto attr
			label attw

> 			`	You follow the lead of the Kimek, and start a run for it up the hill, directly at the fort. Once you're getting closer, however, you see the defenders pouring down what looks like water, and as it flows down the hill to meet your steps, you lose your balance and trip, falling and sliding a bit down the hill. You get up, but try it as you may you cannot find footing with the grass so slippery beneath you, instead having to step on the few tufts that have grown too high for the liquid to cover them. With all their legs, the Kimek aren't so lucky, however, and continue slipping on the floor, with the Saryds in similar trouble. Only the Arachi can press onward, firmly holding onto the dirt beneath the grass with half their legs at a time. They slowly but surely help you and the others move up, and eventually you reach the gate, where the groups that came from the sides are already working on busting it open. You can also see what the Kimek meant when she spoke of "thorns," with various patches of long reddish barbs covering the logs that make up the walls.`

Reviewer 4: ```suggestion
			`	You follow the lead of the Kimek, and start a run for it up the hill, directly at the fort. Once you're getting closer, however, you see the defenders pouring down what looks like water, and as it flows down the hill to meet your steps, you lose your balance and trip, falling and sliding a bit down the hill. You get up, but try it as you may you cannot find footing with the grass so slippery beneath you, instead having to step on the few tufts that have grown too high for the liquid to cover them. With all their legs, the Kimek aren't so lucky, however, and continue slipping on the floor, with the Saryds in similar trouble. Only the Arachi can press onward, firmly holding onto the dirt beneath the grass with half their legs at a time. They slowly but surely help you and the others move up, and eventually you reach the gate, where the groups that came from the sides are already working on busting it open. You can also see what the Kimek meant when she spoke of "thorns," with closely-spaced tufts of long and vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls.`
```

> 			`	You follow the lead of the Kimek and start a run for it up the hill, directly at the fort. Once you get closer, however, you see the defenders pouring down what looks like water. As it flows down the hill to meet your steps, you lose your balance and trip, sliding a bit down the hill. You get up, but, try it as you may, you cannot find footing with the grass, instead having to step on the few tufts that have grown too high for the liquid to cover them. Even with all their legs, the Kimek aren't so lucky and continue slipping on the floor, with the Saryds in similar trouble. Only the Arachi can press onward, firmly holding onto the dirt beneath the grass with half their legs at a time. They slowly but surely help you and the others move up, and eventually you reach the gate, only to find that the groups that came from the sides are already working on busting it open. You can also see what the Kimek meant when she spoke of "thorns," with tufts of vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls.`

Reviewer 6: ```suggestion
			`	You follow the lead of the Kimek and start to run straight up the hill, directly at the fort. Once you get closer, however, you see the defenders, garbed in strips of green cloth as opposed to your blue, pouring out buckets of what looks like water. As it flows down the hill to meet your steps, though, you immediately lose your balance and your feet slip out from under you, and you slide a short distance back down the hill. You get up, but, try it as you may, you cannot find stable footing in the wet the grass, instead having to step on the few tufts that have grown too high for the liquid to cover them. Even with all their legs, the Kimek and Saryds fare no better, and find themselves in similar trouble, losing their balance again and again. Only the Arachi can press onward, digging down into the dirt beneath the grass with half their legs at each step. They slowly but surely help you and the others move up, and eventually you reach the gate, only to find that the groups that came from the sides are already working on busting it open. You can also see what the Kimek meant when she spoke of "thorns;" tufts of vicious, reddish barbs nearly as long as your arm cover the logs that make up the walls.`
```

> 			`	You follow the lead of the Kimek, and start a run for it up the hill, directly at the fort. Once you get closer, however, you see the defenders pouring down what looks like water. As it flows down the hill to meet your steps, you lose your balance and trip, sliding a bit down the hill. You get up, but try it as you may you cannot find footing with the grass, instead having to step on the few tufts that have grown too high for the liquid to cover them. With all their legs, the Kimek aren't so lucky, however, and continue slipping on the floor, with the Saryds in similar trouble. Only the Arachi can press onward, firmly holding onto the dirt beneath the grass with half their legs at a time. They slowly but surely help you and the others move up, and eventually you reach the gate, only to find that the groups that came from the sides are already working on busting it open. You can also see what the Kimek meant when she spoke of "thorns," with tufts of vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls.`

Reviewer: ```suggestion
			`	You follow the lead of the Kimek and start a run for it up the hill, directly at the fort. Once you get closer, however, you see the defenders pouring down what looks like water. As it flows down the hill to meet your steps, you lose your balance and trip, sliding a bit down the hill. You get up, but, try it as you may, you cannot find footing with the grass, instead having to step on the few tufts that have grown too high for the liquid to cover them. Even with all their legs, the Kimek aren't so lucky and continue slipping on the floor, with the Saryds in similar trouble. Only the Arachi can press onward, firmly holding onto the dirt beneath the grass with half their legs at a time. They slowly but surely help you and the others move up, and eventually you reach the gate, only to find that the groups that came from the sides are already working on busting it open. You can also see what the Kimek meant when she spoke of "thorns," with tufts of vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls.`
```
A comma fix, and changed the wording of the sentence describing how the Kimek and Saryds handle the slippery liquid for clarification.

> 			`	You follow the lead of the Kimek and start to run straight up the hill, directly at the fort. Once you get closer, however, you see the defenders, garbed in strips of green cloth as opposed to your blue, pouring out buckets of what looks like water. As it flows down the hill to meet your steps, though, you immediately lose your balance and your feet slip out from under you, and you slide a short distance back down the hill. You get up, but, try it as you may, you cannot find stable footing in the wet the grass, instead having to step on the few tufts that have grown too high for the liquid to cover them. Even with all their legs, the Kimek and Saryds fare no better, and find themselves in similar trouble, losing their balance again and again. Only the Arachi can press onward, digging down into the dirt beneath the grass with half their legs at each step. They slowly but surely help you and the others move up, and eventually you reach the gate, only to find that the groups that came from the sides are already working on busting it open. You can also see what the Kimek meant when she spoke of "thorns;" tufts of vicious, reddish barbs nearly as long as your arm cover the logs that make up the walls.`

Reviewer 3: ```suggestion
			`	You follow the lead of the Kimek and start to run straight up the hill, directly at the fort. Once you get closer, however, you see the defenders, garbed in strips of green cloth as opposed to your blue, pouring out buckets of what looks like water. As it flows down the hill to meet your steps, though, you immediately lose your balance and your feet slip out from under you, and you slide a short distance back down the hill. You get up, but, try it as you may, you cannot find stable footing in the wet grass, instead having to step on the few tufts that have grown too high for the liquid to cover them. Even with all their legs, the Kimek and Saryds fare no better, and find themselves in similar trouble, losing their balance again and again. Only the Arachi can press onward, digging down into the dirt beneath the grass with half their legs at each step. They slowly but surely help you and the others move up, and eventually you reach the gate, only to find that the groups that came from the sides are already working on busting it open. You can also see what the Kimek meant when she spoke of "thorns;" tufts of vicious, reddish barbs nearly as long as your arm cover the logs that make up the walls.`
```

It seems github has trouble highlighting the change:
"the wet the grass" -> "the wet grass"

			choice
				`	(Try and find another way in.)`
					goto attwr
				`	(Help the others bust the gate open.)`

> 			`	You start pushing with the others, the Saryds joining you below, and the Kimek and Arachi crawling up to push the top of the gate. They take turns with others when they get tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it on the gate. It takes some good damage with each time, gradually budging more. The battering ram cannot hit it enough times, however, as the whistle from the start of the event sounds again. It evidently means that time has run out and that the defenders have won.`

Reviewer 6: ```suggestion
			`	You start pushing along with the others, the Saryds joining you from below, and the Kimek and Arachi crawling up to push at the top of the gate. They take turns with each others when one gets tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate finally begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it against the gate. The gate shudders with each hit and gradually begins to give even more, but even the battering ram team cannot break through fast enough; as the whistle from the start of the event sounds again. Evidently this means that time has run out, and the defenders have won.`
```

> 			`	You start pushing with the others, the Saryds joining you below, and the Kimek and Arachi crawling up to push the top of the gate. They take turns with others when they get tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it on the gate. It takes some good damage with each time, gradually budging more. The battering ram cannot it hit enough times, however, as the whistle from the start of the event sounds again. It evidently means that time has run out, and that the defenders have won.`

Reviewer: ```suggestion
			`	You start pushing with the others, the Saryds joining you below, and the Kimek and Arachi crawling up to push the top of the gate. They take turns with others when they get tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it on the gate. It takes some good damage with each time, gradually budging more. The battering ram cannot hit it enough times, however, as the whistle from the start of the event sounds again. It evidently means that time has run out and that the defenders have won.`
```

				goto lost
			label attwr

> 			`	You and a part of the group stick close to the walls and follow along them to the other side, careful not to scratch yourself on the thorns along the fort. You soon find another, smaller gate, and begin pushing. Unlike the main gate, you can feel it budging, and once they realize there aren't many people guarding this gate, the Saryds in the group ask others to move out of the way. You are a bit startled as they suddenly charge at it, almost forcing it open. More defenders come to hold it, though, and the gate becomes increasingly difficult to budge. One Arach heads back to the main group to call for more help, and when they arrive, so do more defenders. The struggle continues for many minutes, with the attackers unable to make a breakthrough. The whistle sounds again, marking the end of the battle, announcing that the defenders have won.`

Reviewer 6: ```suggestion
			`	You and a part of the group stick close to the walls and follow along them to the other side, careful not to scratch yourself on the thorns along the fort. You soon find another, smaller gate, and begin pushing. Unlike what you can see of the efforts you left at the main gate, you can feel this one budging immediately, and there is no slick ground to worry about. Once they realize the vulnerability of this gate, the Saryds in the group ask others to move out of the way. You are a bit startled as they suddenly charge it in unison, almost forcing it open with a single concerted effort. More defenders come to hold it, though, and the gate becomes increasingly difficult to move any further. One Arach heads back to the main group to call for more help, but when they arrive, so do more defenders. The struggle continues for many minutes, with the attackers alongside you unable to make a final breakthrough. The whistle sounds again, marking the end of the battle, announcing that the defenders have won.`
```

> 			`	You and a part of the group stick close to the walls, and follow along them to the other side, careful not to scratch yourself on the thorns along the fort. You soon find another, smaller gate, and begin pushing. Unlike the main gate, you can feel it budging, and once they realize there aren't many people guarding this gate, the Saryds in the group ask others to move out of the way. You are a bit startled as they suddenly charge at it, almost forcing it open. More defenders come to hold it, though, and the gate becomes increasingly difficult to budge. One Arach heads back to the main group to call for more help, and when they arrive so do more defenders. The struggle continues for many minutes, with the attackers unable to make a breakthrough. The whistle sounds again, marking the end of the battle, announcing that the defenders have won.`

Reviewer: ```suggestion
			`	You and a part of the group stick close to the walls and follow along them to the other side, careful not to scratch yourself on the thorns along the fort. You soon find another, smaller gate, and begin pushing. Unlike the main gate, you can feel it budging, and once they realize there aren't many people guarding this gate, the Saryds in the group ask others to move out of the way. You are a bit startled as they suddenly charge at it, almost forcing it open. More defenders come to hold it, though, and the gate becomes increasingly difficult to budge. One Arach heads back to the main group to call for more help, and when they arrive, so do more defenders. The struggle continues for many minutes, with the attackers unable to make a breakthrough. The whistle sounds again, marking the end of the battle, announcing that the defenders have won.`
```

				goto lost
			label attr

> 			`	You follow a group to the edges of this set, losing some time before beginning the run up the hill. Once you finally start making your way up, you turn to see the main group tripping and falling back. It seems the defenders are pouring some slippery liquid down from the battlements above the gate, slowing down those that are coming from that path. You reach the top while the main force still struggles, but your group still makes its way to the gate, to see if you can get it open. On your way, you see what the Kimek meant when she spoke of "thorns," with various patches of long reddish barbs covering the logs that make up the walls. You reach the gate, but as you push and push, it refuses to budge even the slightest bit, having clearly been barricated from the inside. The main group eventually catches up to you, and a lot more people begin pushing now, though it still resists firmly.`

Reviewer 4: ```suggestion
			`	You follow a group to the edges of this set, losing some time before beginning the run up the hill. Once you finally start making your way up, you turn to see the main group tripping and falling back. It seems the defenders are pouring some slippery liquid down from the battlements above the gate, slowing down those that are coming from that path. You reach the top while the main force still struggles, but your group still makes its way to the gate, to see if you can get it open. On your way, you see what the Kimek meant when she spoke of "thorns," with with closely-spaced tufts of long and vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls. You reach the gate, but as you push and push, it refuses to budge even the slightest bit, having clearly been barricated from the inside. The main group eventually catches up to you, and a lot more people begin pushing now, though it still resists firmly.`
```

> 			`	You follow a group to the edges of this set, losing some time before beginning the run up the hill. Once you finally start making your way up, you turn to see the main group tripping and falling back. It seems the defenders are pouring some slippery liquid down from the battlements above the gate, slowing down those that are coming from that path. You reach the top while the main force still struggles, but your group still makes its way to the gate. On your way, you see what the Kimek meant when she spoke of "thorns," with closely-spaced tufts of vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls. You reach the gate, but as you push and push, it refuses to budge even the slightest bit, having clearly been barricaded from the inside. The main group eventually catches up to you and a lot more people begin pushing now, though it still resists firmly.`

Reviewer 6: ```suggestion
			`	You follow a group to the edges of this set, losing some time before beginning your run up the hill. Once you finally start making your way up, you turn to see the main group repeatedly tripping and falling back. It seems the defenders, who are wearing green identifiers instead of blue, are pouring some slippery liquid down from the battlements above the gate, slowing those that are coming along a direct path. You reach the top while the main force still struggles, and your group makes its way to the gate. On your way, you see what the Kimek meant when she spoke of "thorns," with closely-spaced tufts of vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls. You reach the gate, but as you push and push, it refuses to budge even the slightest bit, clearly having been barricaded from the inside. The main group eventually catches up to you, but even with many more people pushing now, the gate still holds firm and unmoving.`
```

> 			`	You follow a group to the edges of this set, losing some time before beginning the run up the hill. Once you finally start making your way up, you turn to see the main group tripping and falling back. It seems the defenders are pouring some slippery liquid down from the battlements above the gate, slowing down those that are coming from that path. You reach the top while the main force still struggles, but your group still makes its way to the gate. On your way, you see what the Kimek meant when she spoke of "thorns," with with closely-spaced tufts of long and vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls. You reach the gate, but as you push and push, it refuses to budge even the slightest bit, having clearly been barricaded from the inside. The main group eventually catches up to you, and a lot more people begin pushing now, though it still resists firmly.`

Reviewer 5: ```suggestion
			`	You follow a group to the edges of this set, losing some time before beginning the run up the hill. Once you finally start making your way up, you turn to see the main group tripping and falling back. It seems the defenders are pouring some slippery liquid down from the battlements above the gate, slowing down those that are coming from that path. You reach the top while the main force still struggles, but your group still makes its way to the gate. On your way, you see what the Kimek meant when she spoke of "thorns," with closely-spaced tufts of vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls. You reach the gate, but as you push and push, it refuses to budge even the slightest bit, having clearly been barricaded from the inside. The main group eventually catches up to you and a lot more people begin pushing now, though it still resists firmly.`
```

			choice
				`	(Try and find another way in.)`
					goto attrr
				`	(Continue trying to bust the gate open.)`
			`	You keep on pushing with the others, the Saryds joining you below, and the Kimek and Arachi crawling up to push the top of the gate, taking turns with others when they get tired. After a while, you hear a cracking noise, and the gate begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it on the gate, which takes some good damage with each time, gradually budging more. It cannot hit enough times, sadly, as the whistle from the start of the event sounds again, meaning that the time has run out, and that the defenders have won.`
				goto lost
			label attrr

> 			`	You and a part of the group stick close to the walls, and follow them until the other side, where you find another, smaller gate, and begin pushing. Unlike the main gate, you can feel it budging, and once they realize there aren't many people guarding this gate, the Saryds in the group ask others to move out of the way, before charging at it and pushing, nearly forcing it open. More defenders come to hold it, though, as it becomes increasingly difficult to budge it. One Arach heads back to the main group to call for more to help, and when they arrive so do more defenders. The struggle continues for a while, before loud bangs are heard from the other side of the fort, and the Kimek organizer comes around the back to tell everyone to return to the main entrance. As you come to the main gate, you see the battering ram has forced the main gate open, something that must've been made easier with so many leaving it unguarded to instead the back gate. With that, the whistle sounds again, marking the end of the battle, and announcing that the attackers have won.`

Reviewer 2: ```suggestion
			`	You and a part of the group stick close to the walls, and follow them until the other side, where you find another, smaller gate, and begin pushing. Unlike the main gate, you can feel it budging, and once they realize there aren't many people guarding this gate, the Saryds in the group ask others to move out of the way, before charging at it and pushing, nearly forcing it open. More defenders come to hold it, though, as it becomes increasingly difficult to budge it. One Arach heads back to the main group to call for more to help, and when they arrive so do more defenders. The struggle continues for a while, before loud bangs are heard from the other side of the fort, and the Kimek organizer comes around the back to tell everyone to return to the main entrance. As you come to the main gate, you see the battering ram has forced the main gate open, something that must've been made easier with so many leaving it unguarded to instead defend the back gate. With that, the whistle sounds again, marking the end of the battle, and announcing that the attackers have won.`
```

Reviewer 2: Added a "defend" near the end of the line.

> 			`	You start pushing with the others, the Saryds joining you below, and the Kimek and Arachi crawling up to push the top of the gate. They take turns with others when they get tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it on the gate. It takes some good damage with each time, gradually budging more. The battering ram cannot it hit enough times, however, as the whistle from the start of the event sounds again. It evidently means that time has run out, and that the defenders have won.`

Reviewer 6: ```suggestion
			`	You start pushing along with the others, the Saryds joining you from below, and the Kimek and Arachi crawling up to push at the top of the gate. They take turns with each others when one gets tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate finally begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it against the gate. The gate shudders with each hit and gradually begins to give even more, but even the battering ram team cannot break through fast enough; as the whistle from the start of the event sounds again. Evidently this means that time has run out, and the defenders have won.`
```

			label won

> 			`	The Kimek organizer in charge of your side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given, and make their way down the hill. Down by the camp, the Kimek who brought you here is waiting to guide your group to many short tables with plates on them; apparently, the reward for winning is a feast. As most of those who participate are Kimek, almost all of the food comes the way they prefer it, a soupy, half-digested mess, but thankfully there are a few meals prepared the way the other species like them, meaning you can pick one prepared for Saryds, which is as close as you could get to human cuisine here.`

Reviewer 4: With all of the slippery liquid involved and the mud that would probably produce, and especially considering that Saryd have hair and a fur coat, and likely Arach have fine hairs for helping sense their environment, it might make sense that there's some kind of communal washing station to clean the worst of the muck off before they get around to trying to eat food.

> 			`	You and a part of the group stick close to the walls, and follow along them to the other side, careful not to scratch yourself on the thorns along the fort. You soon find another, smaller gate, and begin pushing. Unlike the main gate, you can feel it budging, and once they realize there aren't many people guarding this gate, the Saryds in the group ask others to move out of the way. You are a bit startled as they suddenly charge at it, almost forcing it open. More defenders come to hold it, though, and the gate becomes increasingly difficult to budge. One Arach heads back to the main group to call for more help, and when they arrive so do more defenders. The struggle continues for a while, before loud bangs are heard from the other side of the fort. The Kimek organizer comes around the back to tell everyone to return to the main entrance. As you come to the main gate, you see the battering ram has forced the main gate open, something that must've been made easier with so many leaving it unguarded to instead defend the back gate. With that, the whistle sounds again, marking the end of the battle, announcing that the attackers have won.`

Reviewer 6: ```suggestion
			`	You and a part of the group stick close to the walls and follow along them to the other side, careful not to scratch yourself on the thorns along the fort. You soon find another, smaller gate, and begin pushing. Unlike what you can see of the efforts you left at the main gate, you can feel this one budging immediately, and there is no slick ground to worry about. Once they realize the vulnerability of this gate, the Saryds in the group ask others to move out of the way. You are a bit startled as they suddenly charge it in unison, almost forcing it open with a single concerted effort. More defenders come to hold it, though, and the gate becomes increasingly difficult to move any further. One Arach heads back to the main group to call for more help, but when they arrive, so do more defenders. The struggle continues for a while, before you hear loud bangs from the other side of the fort. The Kimek organizer comes around the back to tell everyone to return to the main entrance. As you come to the main gate, you see the battering ram has forced the main gate open, something that must've been made easier with so many leaving it unguarded to instead defend the back gate. With that, the whistle sounds again, marking the end of the battle, announcing that the attackers have won.`
```

> 			`	The Kimek organizer in charge of your side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given, and make their way down the hill, where you're all pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, you head to the camp, where the Kimek who brought you here is waiting to guide your group to many short tables with plates on them; apparently, the reward for winning is a feast. As most of those who participate are Kimek, almost all of the food comes the way they prefer it, a soupy, half-digested mess, but thankfully there are a few meals prepared the way the other species like them, meaning you can pick one prepared for Saryds, which is as close as you could get to human cuisine here.`

Reviewer 4: An actual whole building seems a tad excessive, unless this location is regularly used for this precise purpose. This does work though. :)

Author: Eh, it takes place in a big park, where other similar festivals, or just regular walking around for fun, would be regular activities, I think those justify building some places for visitors to clean themselves up if they need.
Besides, not like the Coalition is struggling with a construction material shortage.

Reviewer 5: You may want to specify the nature of these buildings, though. Are they just washrooms, or showers?

			`	It consists of many cooked vegetables, arranged in a little mount on the plate, with a thin trail of steam still coming from them. Talking with the organizers, you learn that the meals are meant to represent the produce of the fort's "farm," and that those who win earn the right to the banquet. Once you're done eating, you return to where the train dropped you off, and get on the next one to return to the spaceport.`
				decline
			label lost

> 			`	The Kimek organizer of the other side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is aske to return the colored cloth they were given, and make their way down the hill. Down by the camp, the Kimek who brought you here is waiting to guide your opponents to many short tables with plates on them; apparently, the reward for winning is a feast. "A shame it is, that lost the battle your team has," he says. "But hopefully, fun with the event you had."`

Reviewer 2: ```suggestion
			`	The Kimek organizer of the other side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given, and make their way down the hill. Down by the camp, the Kimek who brought you here is waiting to guide your opponents to many short tables with plates on them; apparently, the reward for winning is a feast. "A shame it is, that lost the battle your team has," he says. "But hopefully, fun with the event you had."`
```

> 			`	The Kimek organizer in charge of your side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given, and make their way down the hill, where you're all pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, you head to the camp, where the Kimek who brought you here is waiting to guide your group to many short tables with plates on them; apparently, the reward for winning is a feast. As most of those who participate are Kimek, almost all of the food comes the way they prefer it, a soupy, half-digested mess, but thankfully there are a few meals prepared the way the other species like them, meaning you can pick one prepared for Saryds, which is as close as you could get to human cuisine here.`

Reviewer 5: ```suggestion
			`	The Kimek organizer in charge of your side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given, and make their way down the hill. Your group is pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, you head to the camp, where the Kimek who brought you here is waiting to guide your group to many short tables with plates on them; apparently, the reward for winning is a feast. As most of those who participate are Kimek, almost all of the food comes the way they prefer it, a soupy, half-digested mess, but thankfully there are a few meals prepared the way the other species like them, meaning you can pick one prepared for Saryds, which is as close as you could get to human cuisine here.`
```

> 			`	The Kimek organizer in charge of your side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given and make their way down the hill. Your group is pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, you head to the camp, where the Kimek who brought you here is waiting to guide your group to many short tables with plates on them; apparently, the reward for winning is a feast. As most of those who participate are Kimek, almost all of the food comes the way they prefer it: a soupy, half-digested mess. Thankfully there are a few meals prepared the way the other species like them, meaning you can pick one prepared for Saryds; their meals are the closest you can get to human cuisine here.`

Reviewer 6: ```suggestion
			`	The Kimek organizer in charge of your side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given and make their way down the hill. Your group is pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, you head back to the camp, where the Kimek who brought you here is waiting to guide your group to many short tables with plates on them; apparently, the reward for winning is a feast. As most of those who participate are Kimek, almost all of the food comes the way they prefer it: a soupy, half-digested mess. Thankfully there are a few meals prepared the way the other species like them, meaning you can pick one prepared for Saryds; their meals are the closest you can get to human cuisine here.`
```

> 			`	The Kimek organizer in charge of your side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given, and make their way down the hill. Your group is pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, you head to the camp, where the Kimek who brought you here is waiting to guide your group to many short tables with plates on them; apparently, the reward for winning is a feast. As most of those who participate are Kimek, almost all of the food comes the way they prefer it: a soupy, half-digested mess. Thankfully there are a few meals prepared the way the other species like them, meaning you can pick one prepared for Saryds; their meals are the closest you can get to human cuisine here.`

Reviewer: ```suggestion
			`	The Kimek organizer in charge of your side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given and make their way down the hill. Your group is pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, you head to the camp, where the Kimek who brought you here is waiting to guide your group to many short tables with plates on them; apparently, the reward for winning is a feast. As most of those who participate are Kimek, almost all of the food comes the way they prefer it: a soupy, half-digested mess. Thankfully there are a few meals prepared the way the other species like them, meaning you can pick one prepared for Saryds; their meals are the closest you can get to human cuisine here.`
```

> 			`	Before he leaves with the victors, he explains a bit more about the event, telling you that the meals that will be served at the banquet are meant to represent the produce of the fort's "farm," and that those who win earn the right to it. Finally, he thanks you for participating, and goes with the winners to the tables. You return to where the train dropped you off, and get on the next one to return to the spaceport.`

Reviewer 2: ```suggestion
			`	Before he leaves with the victors, he explains a bit more about the event, telling you that the meals that will be served at the banquet are meant to represent the produce of the fort's 'farm,' and that those who win earn the right to it. Finally, he thanks you for participating, and goes with the winners to the tables. You return to where the train dropped you off, and get on the next one to return to the spaceport.`
```

> 			`	The Kimek organizer in charge of your side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given, and make their way down the hill. Your group is pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, you head to the camp, where the Kimek who brought you here is waiting to guide your group to many short tables with plates on them; apparently, the reward for winning is a feast. As most of those who participate are Kimek, almost all of the food comes the way they prefer it, a soupy, half-digested mess, but thankfully there are a few meals prepared the way the other species like them, meaning you can pick one prepared for Saryds, which is as close as you could get to human cuisine here.`

Reviewer 5: ```suggestion
			`	The Kimek organizer in charge of your side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given, and make their way down the hill. Your group is pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, you head to the camp, where the Kimek who brought you here is waiting to guide your group to many short tables with plates on them; apparently, the reward for winning is a feast. As most of those who participate are Kimek, almost all of the food comes the way they prefer it: a soupy, half-digested mess. Thankfully there are a few meals prepared the way the other species like them, meaning you can pick one prepared for Saryds; their meals are the closest you can get to human cuisine here.`
```

> 			`	It consists of many cooked vegetables, arranged in a little mount on the plate, with a thin trail of steam still coming from them. Talking with the organizers, you learn that the meals are meant to represent the produce of the fort's "farm," and that those who win earn the right to the banquet. Once you're done eating, you return to where the train dropped you off and get on the next one to return to the spaceport.`

Reviewer 6: ```suggestion
			`	The Saryd entrée offered consists of many cooked vegetables, arranged in a little mound on the plate, with a thin trail of steam still coming from them. Talking with the organizers, you learn that the meals are meant to represent the produce of the fort's "farm," and that those who win earn the right to the banquet. Once you're done eating, you return to where the train dropped you off and get on the next one heading back to the spaceport.`
```

> 			`	It consists of many cooked vegetables, arranged in a little mount on the plate, with a thin trail of steam still coming from them. Talking with the organizers, you learn that the meals are meant to represent the produce of the fort's "farm," and that those who win earn the right to the banquet. Once you're done eating, you return to where the train dropped you off, and get on the next one to return to the spaceport.`

Reviewer: ```suggestion
			`	It consists of many cooked vegetables, arranged in a little mount on the plate, with a thin trail of steam still coming from them. Talking with the organizers, you learn that the meals are meant to represent the produce of the fort's "farm," and that those who win earn the right to the banquet. Once you're done eating, you return to where the train dropped you off and get on the next one to return to the spaceport.`
```

> 			`	The Kimek organizer of the other side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given and make their way down the hill. Your group is pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, the Kimek who brought you here is waiting by the camp to guide your opponents to many short tables with plates on them; apparently, the reward for winning is a feast. "A shame it is, that lost the battle your team has," he says. "But hopefully, fun with the event you had."`

Reviewer 6: ```suggestion
			`	The Kimek organizer of the other side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given and make their way down the hill. Your group is pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, the Kimek who brought you here is waiting by the camp to guide your opponents to many short tables with plates on them; apparently, the reward for winning is a feast. "A shame it is, that lost the battle your team has," he says. "But hopefully, still fun with the event you had."`
```

> 			`	The Kimek organizer of the other side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given, and make their way down the hill. Your group is pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, the Kimek who brought you here is waiting by the camp to guide your opponents to many short tables with plates on them; apparently, the reward for winning is a feast. "A shame it is, that lost the battle your team has," he says. "But hopefully, fun with the event you had."`

Reviewer: ```suggestion
			`	The Kimek organizer of the other side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given and make their way down the hill. Your group is pointed to a nearby building to wash off any dirt, if needed. Once you're all cleaned up, the Kimek who brought you here is waiting by the camp to guide your opponents to many short tables with plates on them; apparently, the reward for winning is a feast. "A shame it is, that lost the battle your team has," he says. "But hopefully, fun with the event you had."`
```

> 			`	The Saryd entrée offered consists of many cooked vegetables, arranged in a little mound on the plate, with a thin trail of steam still coming from them. Talking with the organizers, you learn that the meals are meant to represent the produce of the fort's "farm," and that those who win earn the right to the banquet. Once you're done eating, you return to where the train dropped you off and get on the next one heading back to the spaceport.`

Reviewer 3: ```suggestion
			`	The Saryd first course offered consists of many cooked vegetables, arranged in a little mound on the plate, with a thin trail of steam still coming from them. Talking with the organizers, you learn that the meals are meant to represent the produce of the fort's "farm," and that those who win earn the right to the banquet. Once you're done eating, you return to where the train dropped you off and get on the next one heading back to the spaceport.`
```

The game can't display the character 'é'.

> 			`	Before he leaves with the victors, he explains a bit more about the event, telling you that the meals that will be served at the banquet are meant to represent the produce of the fort's "farm," and that those who win earn the right to it. Finally, he thanks you for participating, and goes with the winners to the tables. You return to where the train dropped you off and get on the next one to return to the spaceport.`

Reviewer 6: ```suggestion
			`	Before he leaves with the victors, he explains a bit more about the event, telling you that the meals that will be served at the banquet are meant to represent the produce of the fort's "farm," and that those who win earn the right to it. Finally, he thanks you for participating, and goes with the winners to the tables. You return to where the train dropped you off and get on the next one heading back to the spaceport.`
```

> 			`	Before he leaves with the victors, he explains a bit more about the event, telling you that the meals that will be served at the banquet are meant to represent the produce of the fort's 'farm,' and that those who win earn the right to it. Finally, he thanks you for participating, and goes with the winners to the tables. You return to where the train dropped you off, and get on the next one to return to the spaceport.`

Reviewer: ```suggestion
			`	Before he leaves with the victors, he explains a bit more about the event, telling you that the meals that will be served at the banquet are meant to represent the produce of the fort's "farm," and that those who win earn the right to it. Finally, he thanks you for participating, and goes with the winners to the tables. You return to where the train dropped you off and get on the next one to return to the spaceport.`
```

			choice
				`	(Try and find another way in.)`
					goto attwr
				`	(Help the others bust the gate open.)`

> 			`	You start pushing along with the others, the Saryds joining you from below, and the Kimek and Arachi crawling up to push at the top of the gate. They take turns with each others when one gets tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate finally begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it against the gate. The gate shudders with each hit and gradually begins to give even more, but even the battering ram team cannot break through fast enough; as the whistle from the start of the event sounds again. Evidently this means that time has run out, and the defenders have won.`

Reviewer 7: ```suggestion
			`	You start pushing along with the others, the Saryds joining you from below, and the Kimek and Arachi crawling up to push at the top of the gate. They take turns with each others when one gets tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate finally begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it against the gate. The gate shudders with each hit and gradually begins to give even more, but even the battering ram team cannot break through fast enough as the whistle from the start of the event sounds again. Evidently this means that time has run out, and the defenders have won.`
```

				goto lost
			label attr
			`	You follow a group to the edges of this set, losing some time before beginning your run up the hill. Once you finally start making your way up, you turn to see the main group repeatedly tripping and falling back. It seems the defenders, who are wearing green identifiers instead of blue, are pouring some slippery liquid down from the battlements above the gate, slowing those that are coming along a direct path. You reach the top while the main force still struggles, and your group makes its way to the gate. On your way, you see what the Kimek meant when she spoke of "thorns," with closely-spaced tufts of vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls. You reach the gate, but as you push and push, it refuses to budge even the slightest bit, clearly having been barricaded from the inside. The main group eventually catches up to you, but even with many more people pushing now, the gate still holds firm and unmoving.`
			choice
				`	(Try and find another way in.)`
					goto attrr
				`	(Continue trying to bust the gate open.)`

> 			`	You start pushing along with the others, the Saryds joining you from below, and the Kimek and Arachi crawling up to push at the top of the gate. They take turns with each others when one gets tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate finally begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it against the gate. The gate shudders with each hit and gradually begins to give even more, but even the battering ram team cannot break through fast enough; as the whistle from the start of the event sounds again. Evidently this means that time has run out, and the defenders have won.`

Reviewer 7: ```suggestion
			`	You start pushing along with the others, the Saryds joining you from below, and the Kimek and Arachi crawling up to push at the top of the gate. They take turns with each others when one gets tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate finally begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it against the gate. The gate shudders with each hit and gradually begins to give even more, but even the battering ram team cannot break through fast enough as the whistle from the start of the event sounds again. Evidently this means that time has run out, and the defenders have won.`
```

		conversation
			`A Kimek flags you down when you enter the spaceport. "Busy are you, Captain?" He asks. "Taking place today, a cultural festival is, and participate in it, anyone can. A reenactment of the old 'Spring Siege' folk tale, it is. Play the part of one of the characters, anyone can."`
			choice

> 				`	"That sounds like fun Where is it being held?"`

Reviewer 5: ```suggestion
				`	"That sounds like fun. Where is it being held?"`
```

Reviewer: ha whoops

Reviewer 5: lol no problem
